tools/modules/functions/module_interface.sh

```
@@ -0,0 +1,107 @@
+module_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)